src/main/java/space/npstr/magma/processing/PacketProvider.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2018 Dennis Neufeld
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package space.npstr.magma.processing;
+
+import com.sun.jna.ptr.PointerByReference;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use classes from the Java API instead of Sun classes. ([squid:S1191](https://sonarcloud.io/organizations/napstr-github/rules#rule_key=squid%3AS1191))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=space.npstr.magma%3Amagma&pullRequest=6&issues=AWQs3zm6bXXTsx7OQREj&open=AWQs3zm6bXXTsx7OQREj)

[ISSUE_KEY]: AWQs3zm6bXXTsx7OQREj